presto-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,130 @@
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'PrestoHighTaskFailuresWarning',
+            expr: |||
+              increase(com_facebook_presto_execution_TaskManager_FailedTasks_TotalCount[5m]) > %(alertsHighTaskFailuresWarning)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'The amount of tasks that are failing is increasing, this might affect query processing and could result in incomplete or incorrect results.',
+              description:
+                (
+                  'The number of task failures on {{$labels.instance}} is {{ printf "%%.0f" $value }} which is above the threshold of %(alertsHighTaskFailuresWarning)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'PrestoHighTaskFailuresCritical',
+            expr: |||
+              increase(com_facebook_presto_execution_TaskManager_FailedTasks_TotalCount[5m]) / clamp_min(increase(com_facebook_presto_execution_TaskManager_FailedTasks_TotalCount[10m]), 1) * 100 > %(alertsHighTaskFailuresCritical)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'The amount of tasks that are failing has reached a critical level. This might affect query processing and could result in incomplete or incorrect results.',
+              description:
+                (
+                  'The number of task failures on {{$labels.instance}} is {{ printf "%%.0f" $value }} which is above the threshold of %(alertsHighTaskFailuresCritical)s%%s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'PrestoHighQueuedTaskCount',
+            expr: |||
+              increase(com_facebook_presto_execution_QueryManager_QueuedTaskCount[5m]) > %(alertsHighQueuedTaskCount)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'The amount of tasks that are being put in queue is increasing. A high number of queued tasks can lead to increased query latencies and degraded system performance.',
+              description:
+                (
+                  'The number of queued tasks on {{$labels.instance}} is {{ printf "%%.0f" $value }} which is greater than the threshold of %(alertsHighQueuedTaskCount)s'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'PrestoHighBlockedNodes',
+            expr: |||
+              increase(com_facebook_presto_memory_ClusterMemoryPool_BlockedNodes[5m]) > %(alertsHighBlockedNodesCount)s
```

Comment:
this metric has an additional label on it called `name` which had 2 values - "general" and "reserved". Does that label need to be summed out or maybe included in the description?